jetstream/tests/test_analysis.py

```diff
@@ -31,7 +33,7 @@
 
 
 def _empty_analysis(experiments):
-    x = experiments[0]
+    x: Experiment = experiments[0]
```

Comment:
Mypy?

Reply:
Honestly don't remember, that seems likely but just tested it locally and mypy isn't complaining, so maybe just something that was helpful while I was updating tests?